### ISSUE TITLE
The accuracy rate become lower

### ISSUE THREAD

**Issue by qzhong0605** [onnx/onnx#1710]
When I convert the model from pytorch into caffe2, the top1 and top5 accuracy rate on ILSVRC12 dataset became smaller.

The procedure for converting is:
1.  convert pytorch into onnx IR 
2. convert onnx IR into caffe2 

The model I tested is `resnet18`. The original top1 accuracy rate on `pytorch` is [70.142](https://github.com/Cadene/pretrained-models.pytorch). However,  the converted model on `caffe2` is 0.4994. There is big difference between the model on `pytorch`  and `caffe2`.

One reason I think that cause the decreased accuracy rate is the `image preprocessing`.  I tried to subtract the `mean[0.485*255, 0.456*255, 0.406*255]`  and divide the `std[0.229*255, 0.224*255, 0.225*255]`  , but I got the same low accuracy, which is `0.4994` against the original `70.142`.

I don't know what cause the phenomenon.  Maybe the internal differences between the `pytorch`  and `caffe2` make ?
